distribution/docs/src/main/resources/content/_managing/_running/ingest-command.adoc

```diff
@@ -12,7 +12,8 @@ The ${command-console} has a command-line option for ingesting data.
 [NOTE]
 ====
 Ingesting with the console ingest command creates a metacard in the catalog, but does not copy the resource to the content store.
-The Ingest Command requires read access to the directory being ingested. See the <<{architecture-prefix}url_resource_reader, URL Resource Reader>> for configuring read permission entries to the directory.
+The Ingest Command requires read access to the directory being ingested.
+ <<{architecture-prefix}url_resource_reader, URL Resource Reader>> for configuring read permission entries to the directory.
```

Comment:
❓ `Configure the <<{architecture-prefix}url_resource_reader,URL Resource Reader>> to set read permission entries to the directory.`